src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -325,10 +325,6 @@ public Subject run() {
      * retrieved by this method. After {@code action} is finished, the current
      * subject is reset to its previous value. The current
      * subject is {@code null} before the first call of {@code callAs()}.
-     * <p>
-     * When a new thread is created, its current subject is the same as
```

Comment:
what about change from"when" to "if", in case some implementation still create a new thread?